src/components/Trailer.js

```diff
@@ -0,0 +1,61 @@
+import '../css/Trailer.css'
+import React, { Component } from 'react'
+import endpoints from '../endpoints'
+import VideoSelector from './VideoSelector'
+
+class Trailer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: true,
+      videos: [],
+      selectedTrailer: 0,
+    }
+  }
+
+  componentDidMount = () => {
+    fetch(`${endpoints.movies}/${this.props.id}/videos`)
+      .then(response => response.json())
+      .then(data => this.setState({
+        videos: data.videos
+      }))
+      .then(this.setState({loading: false}))
+      .catch(err => this.setState({
+        error: err.message
+      }))
+  }
```

Comment:
Opened #28 to address the discrepancy between this fetch and the fetch calls in other components.